pkgs/applications/networking/ssb/patchwork/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, appimage-run, fetchurl, runtimeShell, gsettings-desktop-schemas, gtk3, gobject-introspection, wrapGAppsHook }:
```

Comment:
Can we split this? It's a bit long.

Reply:
@asymmetric is there some kind of formatting tool i can run?

Reply:
https://discourse.nixos.org/t/job-implement-a-nix-fmt-formatter/2819

Reply:
aight, so there's a WIP tool but nothing stable or "official"

i have zero interest in debating fmt rules outside the context of automated tooling so i'll just do what you're suggesting